cms/src/main/java/uk/nhs/digital/apispecs/swagger/IsAnyTrueHelper.java

```diff
@@ -0,0 +1,39 @@
+package uk.nhs.digital.apispecs.swagger;
+
+import com.github.jknack.handlebars.Helper;
+import com.github.jknack.handlebars.Options;
+import org.apache.commons.lang.Validate;
+
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.Optional;
+
+public class IsAnyTrueHelper implements Helper<String> {
+
+    public static final IsAnyTrueHelper INSTANCE = new IsAnyTrueHelper();
+
+    public static final String NAME = "isAnyTrue";
+
+    @Override public Object apply(final String items, final Options options) throws IOException {
+
+        Validate.notEmpty(options.params,"One or more input parameters");
+        Validate.noNullElements(options.params,"One or more input parameters");
+        Validate.allElementsOfType(Arrays.asList(options.params), Boolean.class,"Required parameter type is Boolean");
+
+        final Options.Buffer buffer = options.buffer();
+
+        if (hasParams(options)) {
+            buffer.append(options.fn());
+        } else {
+            buffer.append(options.inverse());
+        }
+
+        return buffer;
+    }
+
+    private boolean hasParams(final Options options) {
```

Comment:
This method name is still misleading.

The method checks whether any of the options' parameters is true. The method's name, however, misleadingly communicates something different. As is, it says 'it returns true if options has any parameters at all'.

Reply:
Fixed